src/saga/login.js

```diff
@@ -121,13 +121,21 @@ function composeIncompatibleOVirtApiVersionMessage (oVirtMeta) {
   return `oVirt API version requested >= ${requested}, but ${found} found`
 }
 
+/**
+ * Compare the actual { major, minor } version (JS or ImmutableJS Map) to the required
+ * { major, minor } and return if the **actual** is greater then or equal to **required**.
+ * Backward compatibility of the API is assumed.
+ */
```

Comment:
IMO it's not responsibility of this function to convert data from immutable wrapper. How about using `.toJS()` at place invocation place.

Reply:
It may not be its pure responsibility but it sure is convenient.

Since it's only used in `saga.js`, I'll move it to the helper over there.